astropy/wcs/utils.py

```diff
@@ -1,5 +1,3 @@
-# Licensed under a 3-clause BSD style license - see LICENSE.rst
-
```

Comment:
please don't remove the licence info, neither the empty lines from the docstrings below (sphinx is rather picky for those to be around).

Reply:
Related to this, please don't remove all the empty lines from the file